gammapy/image/utils.py

```diff
@@ -23,6 +27,38 @@
 log = logging.getLogger(__name__)
 
 
+
+def _fftconvolve_wrap(kernel, data):
+    from scipy.signal import fftconvolve
+    return fftconvolve(data, kernel.array, mode='same')
+
+
+def scale_cube_fft(data, kernels, parallel=True):
+    """
+    Compute scale cube by convolving the data with a set of kernels
+    and stack the resulting images along the third axis.
+
+    Parameters
+    ----------
+    data : `~numpy.ndarray`
+        Input data.
+    kernels: list of `~astropy.convolution.Kernel`
+        List of convolution kernels.
+    parallel : bool
+        Whether to use multiprocessing.
```

Comment:
Add `Returns` and maybe mention the shape of `data` and return array?

Also, what about the name?
The main thing this does is to convolve, no? So add "convolve" in?